tools/integration_tests/log_rotation/log_rotation_test.go

```diff
@@ -94,6 +94,7 @@ func TestMain(m *testing.M) {
 	// Set up directory for logs.
 	logDirPath = setup.SetUpLogDirForTestDirTests(logDirName)
 	logFilePath = path.Join(logDirPath, logFileName)
+	setup.SetLogFile(logFilePath)
```

Comment:
Why is this change needed?

Reply:
Because log file set via flag takes precedence now